soql-stdlib/src/main/scala/com/socrata/soql/functions/SoQLFunctions.scala

```diff
@@ -79,6 +79,8 @@ object SoQLFunctions {
   val UnaryPlus = Function("unary +", SpecialFunctions.Operator("+"), Map("a"->NumLike), Seq(VariableType("a")), Seq.empty, VariableType("a"))
   val UnaryMinus = Function("unary -", SpecialFunctions.Operator("-"), Map("a"->NumLike), Seq(VariableType("a")), Seq.empty, VariableType("a"))
 
+  val Magnitude = Function("magnitude", FunctionName("magnitude"), Map("a"->NumLike, "b"->NumLike), Seq(VariableType("a"), VariableType("b")), Seq.empty, VariableType("b"))
```

Comment:
Yes, the result is always positive integer.  But there is value in following the datatype of the primary argument.  Postgres round function does that:

Function: round(dp or numeric)
Return Type: (same as input)
Description: round to nearest integer